src/App.vue

```diff
@@ -1,18 +1,39 @@
 <template>
     <div id="app">
-        <router-view />
+        <main-layout>
+            <template v-slot:sider>
+                <menu-sider />
+            </template>
+            <template v-slot:header>
+                <main-navbar />
+            </template>
+            <template>
+                <main-breadcrum />
+                <router-view />
+            </template>
+        </main-layout>
     </div>
 </template>
 
 <script lang="ts">
 import { Component, Vue, Watch } from 'vue-property-decorator';
-
 import Axios from 'axios';
 
+import MainBreadcrum from '@/components/common/MainBreadcrum.vue';
```

Comment:
@Andresws12 Here I think that you want see "MainBreadcrumb" and the same in all Breadcum should be Breadcrumb in the project